src/modelldcatnotordf/modelldcatno.py

```diff
@@ -968,6 +968,78 @@ def _has_member_to_graph(self, _self: Any) -> None:
             self._g.add((_self, MODELLDCATNO.hasMember, _has_member))
 
 
+class Association(ModelProperty):
+    """A class representing a modelldcatno:Association."""
+
+    __slots__ = "_refers_to"
+
+    _refers_to: ModelElement
+    _identifier: URI
+    _g: Graph
+
+    @property
+    def refers_to(self: Association) -> ModelElement:
+        """Get for refers_to."""
+        return self._refers_to
+
+    @refers_to.setter
+    def refers_to(self: Association, refers_to: ModelElement) -> None:
+        self._refers_to = refers_to
+
+    def to_rdf(
+        self: Association, format: str = "turtle", encoding: Optional[str] = "utf-8"
+    ) -> str:
+        """Maps the role to rdf.
+
+        Args:
+            format: a valid format. Default: turtle
+            encoding: the encoding to serialize into
+
+        Returns:
+            a rdf serialization as a string according to format.
+        """
+        return self._to_graph().serialize(format=format, encoding=encoding)
+
+    def _to_graph(
+        self: Association, type: str = MODELLDCATNO.Association, selfobject: Any = None
```

Comment:
- selfobject er ikke i bruk
- er det mulig å snevre inn typen fra any til "uriref or bnode"

Reply:
Self-objektet er ikke bruk, stemmer det. Men jeg blir nødt til å ha samme signaturen som superklassen: 

`nox > mypy src tests noxfile.py docs/conf.py
src/modelldcatnotordf/modelldcatno.py:1003: error: Signature of "_to_graph" incompatible with supertype "ModelProperty"
Found 1 error in 1 file (checked 24 source files)
nox > Command mypy src tests noxfile.py docs/conf.py failed with exit code 1
`

Reply:
Jeg skal forsøke å snevre inn typen på selfobjektet fra `Any`